gravitee-am-repository/gravitee-am-repository-tests/src/test/java/io/gravitee/am/repository/management/api/DomainRepositoryTest.java

```diff
@@ -101,6 +101,7 @@ private Domain initDomain(String name) {
 
         domain.setAccountSettings(new AccountSettings());
         domain.setLoginSettings(new LoginSettings());
+        domain.getLoginSettings().setResetPasswordOnExpiration(true);
```

Comment:
todo: is it possible to test that this value is read properly by testing it after a find action?